scripts/test_hyde_query_expansion.py

```
@@ -0,0 +1,415 @@
+            # Import search tool dependencies
+            from src.agent.tools.tier1_basic import SearchTool
+            from src.config import get_config, EmbeddingConfig
+            from src.storage import load_vector_store_adapter
+            from src.embedding_generator import EmbeddingGenerator
+            from src.reranker import CrossEncoderReranker
+
+            # Load full config
+            full_config = get_config()
+
+            # Get storage backend
+            storage_dict = full_config.storage if hasattr(full_config.storage, '__dict__') else full_config.storage
```

Ruff (0.14.5)

Useless `if`-`else` condition

(RUF034)